test/utils/mp4-helpers.js

```diff
@@ -109,6 +109,14 @@ module.exports.sampleMoov =
               unityMatrix,
               0x01, 0x2c, 0x00, 0x00, // 300 in 16.16 fixed-point
               0x00, 0x96, 0x00, 0x00), // 150 in 16.16 fixed-point
+          box('edts',
+              box('elst',
+                0x00, // version
+                0x00, 0x00, 0x00, // flags
+                0x00, 0x00, 0x00, 0x01, // entry_count
+                0x00, 0x00, 0x00, 0x00, // segment_duration
+                0x00, 0x00, 0x04, 0x00, // media_time
+                0x00, 0x01, 0x00, 0x00)), // media_rate
```

Comment:
Since it requires a bit more math for the fixed point, it may be worth having a fractional value.

Reply:
good catch. ive changed test value for media rate